libs/internal/include/launchdarkly/network/asio_requester.hpp

```diff
@@ -323,7 +323,7 @@ class AsioRequester {
                 request->Port().value_or(request->Https() ? "https" : "http");
 
             std::shared_ptr<ssl::context> ssl;
```

Comment:
`service` might be something like `9090`, but we still want to use HTTPs if that was the scheme.